mccodec/mc/configuration/serverbound_client_information_config.mbt

```diff
@@ -0,0 +1,61 @@
+///|
+/// Client Information (configuration) packet (0x00)
+/// Sent when player connects or settings change
+struct ClientInformationConfigC2S {
+  locale : String // e.g. "en_GB" (max 16 chars)
+  view_distance : Byte // Client-side render distance in chunks
+  chat_mode : Int // 0: enabled, 1: commands only, 2: hidden
```

Comment:
some minor improvement: add an enum. still, not necessary for this PR but would be nice if we have it